src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +70,12 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      */
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        Predicate<Person> personByAgeAndSexPredicate = person -> person.getAge() >= fromAge
+                && ((person.getSex() == Person.Sex.MAN && person.getAge() <= maleToAge)
+                || (person.getSex() == Person.Sex.WOMAN && person.getAge() <= femaleToAge));
+        return peopleList.stream()
+                .filter(personByAgeAndSexPredicate)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
                .toList();
```
you can call .toList() directly ;)